backend/app/views/spree/admin/adjustment_reasons/edit.html.erb

```diff
@@ -1,11 +1,13 @@
 <%= render 'spree/admin/shared/settings_checkout_tabs' %>
 
 <% content_for :page_title do %>
-  <%= Spree.t(:editing_adjustment_reason) %>
+  <%= Spree.t(:settings) %> &#x2f;
```

Comment:
Instead of writing `&#x2f;` all of the time I think it would be better if we used CSS for this. Something like:

```scss
.page-title > *:before {
  content: ">";
}
```